client/src/partials/projects/projects.html

```diff
@@ -151,58 +152,51 @@
-                    <option value="" disabled> {{ 'SELECT.ENTERPRISE' | translate }} </option>
-                  </select>
-                </div>
+                <label class="control-label">{{ "COLUMNS.ENTERPRISE" | translate }}</label>
+                <select class="form-control" name="enterprise_id" ng-model="ProjectCtrl.project.enterprise_id" id="bhima-enterprise-enterprise_id" ng-options="e.id as e.name for e in ProjectCtrl.enterprises | orderBy:'name'" required>
+                  <option value="" disabled> -- {{ 'SELECT.ENTERPRISE' | translate }} -- </option>
+                </select>
+                <div class="help-block" ng-messages="UpdateForm.enterprise_id.$error" ng-show="UpdateForm.$submitted">
+                  <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+                </div>                
               </div>
 
               <!-- zone de sante for snis -->
               <div class="form-group" ng-class="{ 'has-error' : UpdateForm.$submitted && UpdateForm.zs_id.$invalid }">
-                <label class="col-md-3 control-label">{{ 'COLUMNS.ZS' | translate }}</label>
-                <div class="col-md-9">
-                  <select class="form-control" name="zs_id" ng-model="ProjectCtrl.project.zs_id" id="bhima-project-zs" ng-options="zs.id as zs.format for zs in ProjectCtrl.zones | orderBy:'zone'" required>
-                    <option value="" disabled> -- {{ 'SELECT.ZS' | translate }} -- </option>
-                  </select>
-                </div>
+                <label class="control-label">{{ 'COLUMNS.ZS' | translate }}</label>
+                <select class="form-control" name="zs_id" ng-model="ProjectCtrl.project.zs_id" id="bhima-project-zs" ng-options="zs.id as zs.format for zs in ProjectCtrl.zones | orderBy:'zone'" required>
+                  <option value="" disabled> -- {{ 'SELECT.ZS' | translate }} -- </option>
+                </select>
+                <div class="help-block" ng-messages="UpdateForm.zs_id.$error" ng-show="UpdateForm.$submitted">
+                  <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+                </div>                
               </div>
-
               <!-- end zone de sante for snis -->
-              <div class="form-group">
-                <div class="col-sm-offset-3 col-sm-9">
-                  <div class="checkbox">
-                    <label>
-                      <input type="checkbox" ng-true-value="1" ng-false-value="0" id="bhima-project-locked"  name="locked" ng-model="ProjectCtrl.project.locked"> {{ 'COLUMNS.LOCKED' | translate }}
-                    </label>
-                  </div>
-                </div>
+              
+              <div class="form-group" ng-class="{ 'has-error' : UpdateForm.$submitted && UpdateForm.locked.$invalid }">
+                <label class="control-label">
+                  <input type="checkbox"  name="locked" id="locked" ng-true-value="1" ng-false-value="0" 
```

Comment:
I've noticed that one end to end test fails on this branch:

> 1) Projects Modules successfully edits an project:
       NoSuchElementError: No element found using locator: By.id("bhima-project-locked")

I think this is a simple case of ensuring the new improved input has the same id! Once this is done you can verify that all tests pass.